var/spack/repos/builtin/packages/gdb/package.py

```diff
@@ -68,6 +68,7 @@ class Gdb(AutotoolsPackage, GNUMirrorPackage):
     # Optional dependencies
     depends_on('python', when='+python', type=('build', 'link', 'run'))
     depends_on('python@:3.6', when='@:8.1+python', type=('build', 'link', 'run'))
+    depends_on('python@:3.8', when='@:9.2+python', type=('build', 'link', 'run'))
```

Comment:
Can you add a comment above this directive with the link from the bug in your PR description?

Reply:
Ok 👌